openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx

```diff
@@ -109,47 +115,18 @@ const GlossaryPageV1 = () => {
     setExpandedKey([data.name]);
   };
 
-  const fetchGlossaryList = (paging = '') => {
-    setIsLoading(true);
-    getGlossariesWithRootTerms(paging, 100, ['owner', 'tags', 'reviewers'])
-      .then((data: ModifiedGlossaryData[]) => {
-        if (data?.length) {
-          setGlossaries(data);
-          setGlossariesList(data);
-          initSelectGlossary(data[0]);
-        } else {
-          setGlossariesList([]);
-        }
-        setIsLoading(false);
-      })
-      .catch((err: AxiosError) => {
-        showToast({
-          variant: 'error',
-          body: err.response?.data?.message ?? 'Something went wrong!',
-        });
-        setIsLoading(false);
-      })
-      .finally(() => {
-        handleChildLoading(false);
-      });
-  };
-
   const fetchGlossaryTermByName = (
-    name: string,
-    pos: string[],
-    key?: string
+    fqn: string,
+    pos: number[],
+    arrGlossary: ModifiedGlossaryData[]
```

Comment:
Can you please add comments to describe the method and its arguments?

Reply:
Done